ui/src/shared/components/StaticLegend.js

```diff
@@ -0,0 +1,120 @@
+import React, {PropTypes, Component} from 'react'
+import _ from 'lodash'
+import uuid from 'node-uuid'
+
+const style = {
+  position: 'absolute',
+  width: 'calc(100% - 32px)',
+  bottom: '8px',
+  left: '16px',
+  paddingTop: '8px',
+}
+
+const removeMeasurement = (label = '') => {
```

Comment:
Can we add a comment here to give an example of the strings we'll be matching against?

Reply:
moved to `shared/graphs/helpers` and commented